### PR TITLE
fix(emails): add post change email template

### LINKED ISSUE #2153: User should receive an email when primary email changed
From feature doc https://docs.google.com/document/d/1SaJEOGi2GEd69-Rype480Syl7DPdCSFSfRxmS7oKUrE/edit?ts=59d462e3

* User should receive a notification to the new primary email address that this is the new primary email address

### PR DESCRIPTION
**Pull request by vbudhram**
WIP that fixes #2153 

- [x] Add tests

### CODE REVIEW

**Comment by vbudhram** [comment]
@mozilla/fxa-devs low priority r?

File: lib/senders/templates/_versions.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "passwordChangedEmail": 1,
   "passwordResetEmail": 1,
   "passwordResetRequiredEmail": 1,
+  "postChangePrimaryEmail": 1,
```
Comment (philbooth) [inline lib/senders/templates/_versions.json]: This time did it work??

Comment (vbudhram) [reply lib/senders/templates/_versions.json]: Script worked, but I ran it manually after adding templates.

**Comment by vbudhram** [comment]
@philbooth Thanks!

**Pull request merged**